src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse42.cs

```diff
@@ -18,115 +18,242 @@ public static class Sse42
         /// <summary>
         /// int _mm_cmpistra (__m128i a, __m128i b, const int imm8)
         ///   PCMPISTRI xmm, xmm/m128, imm8
-        /// int _mm_cmpistrc (__m128i a, __m128i b, const int imm8)
-        ///   PCMPISTRI xmm, xmm/m128, imm8
-        /// int _mm_cmpistro (__m128i a, __m128i b, const int imm8)
-        ///   PCMPISTRI xmm, xmm/m128, imm8
-        /// int _mm_cmpistrs (__m128i a, __m128i b, const int imm8)
+        /// </summary>
+        public static bool CompareImplicitLengthNotCAndNotZ(Vector128<sbyte> left, Vector128<sbyte> right, StringComparisonMode mode) => CompareImplicitLengthNotCAndNotZ(left, right, mode);
```

Comment:
Do we really need the words `ImplicitLength` and `ExplicitLength` in the method names?  Isn't the fact that I used the overload without specifying the lengths enough to disambiguate that I wanted to use implicit lengths?

Reply:
Good point! Will try to eliminate these two words.